ideascube/monitoring/tests/test_views.py

```diff
@@ -424,7 +521,23 @@ def test_cannot_loan_twice_same_item(staffapp, user):
     assert Loan.objects.due().count() == 1
 
 
-def test_can_loan_twice_same_item_when_first_is_returned(staffapp, user):
+def test_cannot_loan_twice_same_item_from_serial(staffapp, user):
```

Comment:
How about an item loaned by barcode, can we then loan it a second time by serial? 😜